ballerina/tests/test.bal

```diff
@@ -1071,16 +1070,16 @@ function testDeleteQueue() returns error? {
 }
 isolated function testDeleteNonExistentQueue() returns error? {
     string realQueueUrl = check sqsClient->createQueue("test-delete");
-    log:printInfo("realQueueUrl", url=realQueueUrl);
     string queueUrl = realQueueUrl + "-non-existent-queue";
-    log:printInfo("queueUrl",url=queueUrl);
     Error? result = sqsClient->deleteQueue(queueUrl);
-    test:assertTrue(result is Error, "Expected uncessfull deletion.");
+    test:assertTrue(result is Error, "Expected unsuccessful deletion.");
     if result is error {
         ErrorDetails details = result.detail();
         test:assertEquals(details.errorCode, "AWS.SimpleQueueService.NonExistentQueue");
         test:assertEquals(details.httpStatusCode, 400);
         test:assertEquals(details.errorMessage, "The specified queue does not exist.");
+        string errMsg = <string>details.errorMessage;
```

Comment:
```suggestion
        string errMsg = check details.errorMessage.ensuretype();
```

Use `ensuretype` instead of explicit casting: https://learn-ballerina.github.io/best_practices/avoid_unnecessary_type_casts.html?highlight=cast#avoid-unnecessary-type-casts